app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -239,62 +245,68 @@ def create_custom_button_hash(input, record, options = {})
       :enabled   => options[:enabled],
       :klass     => ApplicationHelper::Button::ButtonWithoutRbacCheck,
       :url       => "button",
-      :url_parms => "?id=#{record.id}&button_id=#{button_id}&cls=#{record.class}&pressed=custom_button&desc=#{button_name}"
+      :url_parms => "?id=#{record_id}&button_id=#{button_id}&cls=#{model}&pressed=custom_button&desc=#{button_name}"
```

Comment:
@vecerek .. Aand this will need `send_checked`  (https://github.com/ManageIQ/manageiq-ui-classic/pull/642)